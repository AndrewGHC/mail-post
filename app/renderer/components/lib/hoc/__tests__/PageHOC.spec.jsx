import React from 'react';
import { shallow } from 'enzyme';

import pageHoc from '../pageHoc';

const TestComponent = () => <div>hello</div>;
describe('pageHoc', () => {
  it('matches snapshot', async () => {
    const Component = pageHoc(TestComponent);
    const wrapper = shallow(<Component />);
    expect(wrapper).toMatchSnapshot();
  });
});
