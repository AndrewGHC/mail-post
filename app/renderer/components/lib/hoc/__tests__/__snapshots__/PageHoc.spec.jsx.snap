// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageHoc matches snapshot 1`] = `
<PageHoc__StyledPageHOC>
  <TestComponent />
</PageHoc__StyledPageHOC>
`;
